recipes/texlive-core/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set version = "20150521" %}
+
+package:
+    name: texlive-core
+    version: {{ version }}
+
+source:
+    fn: texlive-{{ version }}-source.tar.xz
+    url: http://mirrors.ctan.org/systems/texlive/Source/texlive-{{ version }}-source.tar.xz
+    sha256: ed9bcd7bdce899c3c27c16a8c5c3017c4f09e1d7fd097038351b72497e9d4669
+
+build:
+    number: 0
+    skip: True  # [win]
+    detect_binary_files_with_prefix: true
+
+requirements:
+    build:
+        - pkg-config
+        - cairo
+        - gmp
+        - icu
+        - mpfr
+        - pixman
+        - harfbuzz
```

Comment:
Apparently this is Linux only. Let's add a selector so we can deal with that until this changes.